auto_tests/save_load_test.c

```diff
@@ -16,6 +16,16 @@
 #include "../toxcore/util.h"
 #include "check_compat.h"
 
+/* The Travis-CI container responds poorly to ::1 as a localhost address
+ * You're encouraged to -D FORCE_TESTS_IPV6 on a local test  */
+#ifdef FORCE_TESTS_IPV6
+#define TOX_LOCALHOST "::1"
+#else
+#define TOX_LOCALHOST "127.0.0.1"
+#endif
+
+#define TCP_RELAY_PORT 33430
```

Comment:
Perhaps add an ifdef/undef/endif for this, or rename it. All symbols including macros must be globally unique. In the case of macros, you can undef before you define it to keep the name and redefine it.